controllers/storagecluster/cephcluster.go

```diff
@@ -116,9 +117,17 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	platform, err := r.platform.GetPlatform(r.Client)
+	if err != nil {
+		r.Log.Error(err, "Failed to get platform")
+	} else if platform == v1.IBMCloudPlatformType || platform == IBMCloudCosPlatformType {
+		r.Log.Info("Increasing Mon failover timeout to 15m for %s", platform)
```

Comment:
Log usage is not correct.  Its `Info(msg string, keysAndValues ...interface{})`.  You can check other examples in OCS for reference.

Reply:
Changed